areas-of-d-usage.dd

```diff
@@ -322,4 +322,7 @@ Macros:
                 margin-top: -200px;
             }
         }
+        h2 {
```

Comment:
I disagree with you two, but I doubt I'll be able to change your minds and it's not really that big of an issue to me :)